### DISCUSSION TITLE
Safari Plus Settings can't be Selected and nothing happens

### DISCUSSION THREAD

**Discussion by MikeyTadd** [standardnotes/forum#2548] [General]
**Describe the bug**
Just updated to Plus, and using Safari on macOS I can't select some of the plus features, such as the code editor or to change the theme. 

**To Reproduce**
1. Login to account in Safari2. 
2. Change the appearance, selecting any of the options closes menu but nothing changes3. 
3. Change the editor, selecting anything away from the visible list doesn't do anything (i.e. the code editor). 

**Expected behavior**
Colour settings change and code editor can be selected

**Screenshots**

**Desktop (please complete the following information):**
 - OS: macOS
 - Browser Safari
 - Version 15.3 (17612.4.9.1.8)

**Additional context**
Many of the paid settings can't be selected, if using another device (i.e. desktop client or mobile) they display correctly. It seems to be most of the plus settings which are not displayed by default and you have to scroll to select them.

**Comment by atmoio** [comment]
Hmm, indeed, I can replicate this with some of the menus. It looks like they close before the interaction is registered. We'll look into this, but for now, please use either the desktop app, or another browser.
